addons/mail/static/src/models/activity_group.js

```diff
@@ -60,7 +60,7 @@ registerModel({
     },
     onChanges: [
         {
-            dependencies: ['total_count', 'type'],
+            dependencies: ['total_count', 'type', 'overdue_count'],
```

Comment:
Why isn't the dependency `planned_count` since it is what is used inside the method?